src/cargo/util/toml/mod.rs

```diff
@@ -1021,6 +1021,8 @@ impl TomlManifest {
             None => false,
         };
 
+        let warn_default_feat = summary.features().contains_key("default-features");
```

Comment:
I don't think the flag is necessary.  You can push directly onto the `warnings` vec here.

Reply:
Whoops, you're right. Thanks!